package.json

```diff
@@ -25,7 +25,12 @@
     "update": "ng update @nrwl/workspace",
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
-    "help": "nx help"
+    "help": "nx help",
+    "build:schematics": "tsc -p tsconfig.schematics.json",
+    "copy:schemas": "copyfiles  schematics/*/schema.json dist/libs/angular-routing/",
```

Comment:
We don't need an external package for this. ng-packagr has an `assets` property in the `ng-package.json` to add globs for files.